src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -35,7 +36,8 @@ def acr_import(cmd,
                target_tags=None,
                resource_group_name=None,
                repository=None,
-               force=False):
+               force=False,
+               no_wait=False):
```

Comment:
@CarolineNB please add the comment `# pylint: disable=too-many-locals` to suppress the style warning "Too many local variables". Since these variables are command parameters, we can ignore this error. Please reference [acr_task_create](https://github.com/Azure/azure-cli/blob/aaffbc5cafa4535aadc180136ec094f1725a0f38/src/azure-cli/azure/cli/command_modules/acr/task.py#L43).